src/inc/corcompile.h

```diff
@@ -1823,4 +1824,13 @@ inline bool IsReadyToRunCompilation()
 #endif
 }
 
+inline bool IsLargeVersionBubbleEnabled()
```

Comment:
This method should be ever called at crossgen time only. It should be never called at runtime.

Could you please to move the whole method under ifdef to let compiler verify it?


```
#ifdef FEATURE_READYTORUN_COMPILER
inline bool IsLargeVersionBubbleEnabled()
{
    return g_fLargeVersionBubble;
}
#endif
```

Reply:
Done! (this actually dug out a piece of dead code)